audio/param.rs

```diff
@@ -256,25 +256,26 @@ pub enum RampKind {
     Exponential,
 }
 
-#[derive(Clone, Copy, PartialEq, Debug)]
+#[derive(Clone, PartialEq, Debug)]
 /// https://webaudio.github.io/web-audio-api/#dfn-automation-event
 pub(crate) enum AutomationEvent {
     SetValue(f32),
     SetValueAtTime(f32, Tick),
     RampToValueAtTime(RampKind, f32, Tick),
     SetTargetAtTime(f32, Tick, /* time constant, units of Tick */ f64),
+    SetValueCurveAtTime(Vec<f32>, Tick, /* duration */ Tick),
```

Comment:
both these variants should have a comment saying that the first `Tick` is the start time